srcpkgs/python3-pywlroots-0.15/template

```diff
@@ -0,0 +1,15 @@
+# Template file for 'python3-pywlroots-0.15'
+pkgname=python3-pywlroots-0.15
```

Comment:
I'd name it `python3-pywlroots0.15` instead, as it's more in line with how other versioned packages are named.

Reply:
This naming schema is easier to manipulate, though.